compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/BallerinaParserErrorHandler.java

```diff
@@ -558,6 +559,12 @@
     private static final ParserRuleContext[] NAMED_ARG_MATCH_PATTERN_RHS =
             { ParserRuleContext.NAMED_ARG_MATCH_PATTERN, ParserRuleContext.REST_MATCH_PATTERN };
 
+    private static final ParserRuleContext[] ORDER_KEY_LIST_START =
```

Comment:
since this list has only one member, we can remove the list and directly use `EXPRESSION` as the next rule, where ORDER_KEY_LIST_START is used. Can remove it from hasAlternativePaths() methods also, since it has only one path